saleor/order/models.py

```diff
@@ -49,11 +49,12 @@ def ready_to_capture(self):
         """Return orders with payments to capture.
 
         Orders ready to capture are those which are not draft or canceled and
-        have a preauthorized payment.
+        have a preauthorized payment. The preauthorized payment can not
+        already be partially or fully captured.
         """
         qs = self.filter(
-            payments__is_active=True, payments__charge_status__in=[
-                ChargeStatus.NOT_CHARGED, ChargeStatus.CHARGED])
+            payments__is_active=True,
+            payments__charge_status=ChargeStatus.NOT_CHARGED)
```

Comment:
Shouldn't that include partially charged orders as well? They have still some amount to capture.

Reply:
I'm not sure about braintree, in stripe if you authorize 10USD but the staff only capture 8USD, the rest 2USD will be automatically released and can not be captured anymore. So in this case, the payment which has already been captured partially can not be captured anymore...

What about braintree?

Reply:
Update: If we decide not to support **multple partial capture** as like implementing ```submit_for_partial_settlement``` in braintree, partially charged orders won't have any amount to capture since the rest amount is released automatically.